examples/js/loaders/GLTFLoader.js

```diff
@@ -1669,6 +1669,8 @@ THREE.GLTFLoader = ( function () {
 
 			onLoad( result );
 
+			parser.cache.removeAll();
```

Comment:
What's this?

Reply:
The gltf parser also keeps a cache of things while its loading, don't see any reason we would want to keep it around after load, and it was retaining things.